src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeFormatInfoScanner.cs

```diff
@@ -112,33 +112,33 @@ internal class DateTimeFormatInfoScanner
         // The collection fo date words & postfix.
         internal List<string> m_dateWords = new List<string>();
         // Hashtable for the known words.
-        private static volatile Dictionary<string, string>? s_knownWords;
+        private static volatile HashSet<string>? s_knownWords;
 
-        private static Dictionary<string, string> KnownWords =>
+        private static HashSet<string> KnownWords =>
             s_knownWords ??=
```

Comment:
This is the only static in the whole type. It does not really need to use manual lazy initialization.

Reply:
Cool, will change.